userspace/libsinsp/plugin.h

```diff
@@ -28,6 +28,10 @@ limitations under the License.
 
 #include "filter_check_list.h"
 
+#ifndef _WIN32
+#define HINSTANCE void*
+#endif
+
```

Comment:
How about

```
#ifdef _WIN32
typedef HINSTANCE sinsp_plugin_handle;
#else 
typedef void* sinsp_plugin_handle;
#endif 
```

and then using `sinsp_plugin_handle` in place of `HINSTANCE`?

Reply:
That's a very good idea!